### ISSUE TITLE
Question: Do core nodes handle MQTT connections?

### ISSUE THREAD

**Issue by alwyngo** [emqtt/emq-docs-cn#2367]
<https://www.emqx.io/docs/zh/latest/deploy/cluster/mria-introduction.html>
Documentation version v5.5

Based on business requirements, core nodes can connect to MQTT clients or be used purely as the cluster's database. We recommend:

In small clusters (3 nodes or fewer), there's no need to use the Core + Replicant replication mode; instead, let the core nodes handle all connections.
In very large clusters (10 nodes or more), core nodes do not handle MQTT connections, which provides better stability and horizontal scalability.
In medium-sized clusters, whether core nodes handle MQTT connections depends on many factors, and you need to test based on your actual scenario to know which option is better.

Question: In a cluster with both core nodes and replicant nodes, can core nodes be configured to accept connections or not accept connections? Or do core nodes not handle MQTT connections at all when replicant nodes are present?

The document <https://docs.emqx.com/zh/emqx-operator/2.1.0/tasks/configure-emqx-core-replicant.html#%E9%85%8D%E7%BD%AE-emqx-%E9%9B%86%E7%BE%A4-core-%E8%8A%82%E7%82%B9> states:
If the EMQX cluster is configured with Replicant nodes, MQTT client requests will connect to Replicant nodes; otherwise, they will connect to Core nodes. Requests to access the EMQX Dashboard will only connect to Core nodes.

---
*This issue was automatically translated from Chinese.*

**Comment by id** [comment]
Yes, core nodes can handle MQTT traffic as well. But if you are using emqx-operator, and have replicant nodes, MQTT traffic will go to replicant nodes only.
